src/deluge/dsp/filter/hpladder.cpp

```diff
@@ -85,7 +85,7 @@ void HpLadderFilter::doFilterStereo(q31_t* startSample, q31_t* endSample) {
 		currentSample += 1;
 	} while (currentSample < endSample);
 }
-inline q31_t HpLadderFilter::doHPF(q31_t input, HPLadderState& state) {
+inline q31_t __attribute__((always_inline)) HpLadderFilter::doHPF(q31_t input, HPLadderState& state) {
```

Comment:
we should probably prefer the standards-compliant syntax for this:

```
[[gcc::always_inline]]
```

instead of

```
__attribute__((always_inline))
```